README.md

```diff
@@ -1,2 +1,12 @@
-# user-microservice
-User management microservice for our online game framework. 
+# microservice-user
+Microservice to manage users.
+
+Beginning of code was based off of Nic Jackson's building microservices in Go : https://www.youtube.com/playlist?list=PLmD8u-IFdreyh6EUfevBcbiuCKzFk0EW_
```

Comment:
Mettre des commentaires pertinents à ce microservice. C'est quoi les endpoints?
On va les mettre dans le Read the docs éventuellement, mais il faut le faire